File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.c
int removeDuplicates(int* nums, int numsSize) {
    int i,j=1;
    
   
    for(i=1;i<numsSize;i++)
    {
        if(nums[i-1]!= nums[i])
        {
            nums[j] = nums[i];
            j++;
        }
        
    }
    return j;
    
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        HashSet<Integer> h = new HashSet<>();
        int j=1,i;
        for(i=1;i<nums.length;i++)
        {
           if(nums[i-1] != nums[i])
           {
            nums[j] = nums[i];
            j++;
           }
        }
       return j;
    }
}